resources/telemetry/values.yaml

```diff
@@ -18,6 +18,7 @@ global:
       name: "fluent-bit"
       version: "2.1.2-95ba1b02"
       directory: "prod/tpi"
-  highPriorityClassName: "kyma-system-priority"
+  highPriorityClassName: "telemetry-priority-class-high"
+  PriorityClassName: "telemetry-priority-class"
```

Comment:
Let's move this out of the global section since it's specific for the telemetry component now. The templates should deal with both a global and a local value. Please double-check.